conf/app.routes

```diff
@@ -36,7 +36,7 @@ GET    /determine-eligibility                               controllers.Determin
 GET    /not-enrolled                                        controllers.NotEnrolledController.notEnrolled
 GET    /not-vat-registered                                  controllers.NotEnrolledController.notVatRegistered
 GET    /request-access-to-self-assessment                   controllers.NotEnrolledController.notSaEnrolled
-GET    /no-nino-found-sia                                   controllers.NotEnrolledController.simpNoNino
+GET    /no-nino-found-simp                                  controllers.NotEnrolledController.simpNoNino
```

Comment:
this is going to disappear eventually anyway I think so we don't technically need this change

Reply:
fair